cmd/applyconfig/applyconfig_test.go

```diff
@@ -105,7 +107,7 @@ func TestMakeOcCommand(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			cmd := makeOcCommand(tc.cmd, tc.path, tc.user)
 			if !reflect.DeepEqual(cmd.Args, tc.expected) {
-				t.Errorf("Command differs from expected:\n%s", diff.ObjectReflectDiff(tc.expected, cmd.Args))
+				t.Errorf("Command differs from expected:\n%s", cmp.Diff(tc.expected, cmd.Args))
```

Comment:
Why did we change this?

Reply:
`// DEPRECATED: use github.com/google/go-cmp/cmp.Diff`, I assume (https://github.com/kubernetes/apimachinery/blob/master/pkg/util/diff/diff.go#L70)

Reply:
Thanks Petr.